### ISSUE TITLE
StyleManager not pulling the updated parameters after updating

### ISSUE THREAD

**Issue by rcloss** [artf/grapesjs-mjml#235]
I'm using the most recent MJML and GrapesJS and have set up a simple test HTML file using the grapesjs mjml as a base (I remove toastr and other things so the only javascript loading is jquery, grapesjs and the mjml/ckeditor plugins. 

Here's an example of what's happening

I start with a clean empty canvas, drag in a 1 column block which has an mj-text in it by default, I click on the mj-text, go to the StyleManager and it has a top padding of 10px, I change it to 25px and the change works, but if I click away and come back even though it's still 25px and if I view source I can see that 25px, in the StyleManager itself it displays 10px in the "Top Padding" box

it seems to be working fine in the demo so I don't know what's going on.

EDIT: I switched through the older versions I still had installed, and it looks like it was working in 0.2.4 and not working as of 0.4.1

Here's the code of my tester html if it helps, but it's all pretty boilerplate

`<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>GrapesJS Demo - Free Open Source MJML Newsletter Builder</title>
    <meta name="description" content="GrapesJS MJML Newsletter Builder - Best Free Open Source Responsive MJML Newsletter Editor">
    <meta name="keywords" content="newsletter builder, web template editor, html builder, html editor">
    <link rel="stylesheet" href="./js/grapesjs-dev-0.16.41/dist/css/grapes.min.css?v0.16.41">

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="./js/grapesjs-dev-0.16.41/dist/grapes.min.js"></script>
	<script src="./js/ckeditor/ckeditor.js"></script>
    
    <script src="./js/grapesjs-plugin-ckeditor-master/dist/grapesjs-plugin-ckeditor.min.js"></script>
    <script src="./js/grapesjs-mjml-master-0.5.2/dist/grapesjs-mjml.min.js?v=0.5.2"></script>

  </head>
  <style>
    .gjs-clm-tags {
       display: none;
    }

    .gjs-four-color {
      color: #f45e43;
    }

    .gjs-logo-version {
      background-color: #4b4e54;
    }
  </style>
  <body>

    <div id="gjs">

      <mjml><mj-body>
        <mj-section background-color="#f0f0f0">
          <mj-column>
            <mj-text font-style="italic" font-size="20px" color="#626262">
              My Company
            </mj-text>
          </mj-column>
        </mj-section>

        <mj-section background-url="//1.bp.blogspot.com/-TPrfhxbYpDY/Uh3Refzk02I/AAAAAAAALw8/5sUJ0UUGYuw/s1600/New+York+in+The+1960's+-+70's+(2).jpg" background-size="cover" background-repeat="no-repeat">
          <mj-column>
            <mj-text align="center" color="#fff" font-size="40px" font-family="Helvetica Neue">Slogan here</mj-text>
            <mj-button background-color="#F63A4D" href="#">
              Promotion
            </mj-button>
          </mj-column>
        </mj-section>

        <mj-section background-color="#fafafa">
          <mj-column width="400px">
            <mj-text font-style="italic" font-size="20px" font-family="Helvetica Neue" color="#626262">My Awesome Text</mj-text>
            <mj-text color="#525252">
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin rutrum enim eget magna efficitur, eu semper augue semper. Aliquam erat volutpat. Cras id dui lectus. Vestibulum sed finibus lectus, sit amet suscipit nibh. Proin nec commodo purus. Sed eget
              nulla elit. Nulla aliquet mollis faucibus.
            </mj-text>
            <mj-button background-color="#F45E43" href="#">Learn more</mj-button>
          </mj-column>
        </mj-section>

        <mj-section background-color="white">
          <mj-column>
            <mj-image width="200px" src="https://designspell.files.wordpress.com/2012/01/sciolino-paris-bw.jpg" />
          </mj-column>
          <mj-column>
            <mj-text font-style="italic" font-size="20px" font-family="Helvetica Neue" color="#626262">
              Find amazing places
            </mj-text>
            <mj-text color="#525252">
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin rutrum enim eget magna efficitur, eu semper augue semper. Aliquam erat volutpat. Cras id dui lectus. Vestibulum sed finibus lectus.</mj-text>
          </mj-column>
        </mj-section>

        <mj-section background-color="#fbfbfb">
          <mj-column>
            <mj-image width="100px" src="//191n.mj.am/img/191n/3s/x0l.png" />
          </mj-column>
          <mj-column>
            <mj-image width="100px" src="//191n.mj.am/img/191n/3s/x01.png" />
          </mj-column>
          <mj-column>
            <mj-image width="100px" src="//191n.mj.am/img/191n/3s/x0s.png" />
          </mj-column>
        </mj-section>

        <mj-section background-color="#e7e7e7">
          <mj-column>
            <mj-button href="#">Hello There!</mj-button>
            <mj-social font-size="15px" icon-size="30px" mode="horizontal">
              <mj-social-element name="facebook" href="https://mjml.io/">
                Facebook
              </mj-social-element>
              <mj-social-element name="google" href="https://mjml.io/">
                Google
              </mj-social-element>
              <mj-social-element name="twitter" href="https://mjml.io/">
                Twitter
              </mj-social-element>
            </mj-social>
          </mj-column>
        </mj-section>
      </mj-body></mjml>

    </div>

    <script type="text/javascript">
      CKEDITOR.dtd.$editable.a = 1;

      var lp = './img/';
      var plp = '//placehold.it/350x250/';
      var images = [
        plp+'78c5d6/fff/image1.jpg', plp+'459ba8/fff/image2.jpg', plp+'79c267/fff/image3.jpg',
        plp+'c5d647/fff/image4.jpg', plp+'f28c33/fff/image5.jpg', plp+'e868a2/fff/image6.jpg', plp+'cc4360/fff/image7.jpg',
        lp+'work-desk.jpg', lp+'phone-app.png', lp+'bg-gr-v.png'
      ];
      // Set up GrapesJS editor with the MJML plugin
      var editor = grapesjs.init({
        clearOnRender: true,
        height: '100%',
        avoidInlineStyle: false,
        storageManager: {id: 'gjs-mjml-'},
        //storageManager:{ autoload: 0 },
        assetManager: {
          assets: images,
          upload: 0,
          uploadText: 'Uploading is not available in this demo',
        },
        container : '#gjs',
        fromElement: true,

        plugins: ['grapesjs-mjml', 'gjs-plugin-ckeditor'],
        pluginsOpts: {
          'gjs-plugin-ckeditor': {
            position: 'center',
            options: {
              startupFocus: true,
              extraAllowedContent: '*(*);*{*}', // Allows any class and any inline style
              allowedContent: true, // Disable auto-formatting, class removing, etc.
              enterMode: CKEDITOR.ENTER_BR,
              extraPlugins: 'sharedspace,justify,colorbutton,panelbutton,font',
              toolbar: [
                { name: 'styles', items: ['Font', 'FontSize' ] },
                ['Bold', 'Italic', 'Underline', 'Strike'],
                {name: 'paragraph', items : [ 'NumberedList', 'BulletedList']},
                {name: 'links', items: ['Link', 'Unlink']},
                {name: 'colors', items: [ 'TextColor', 'BGColor' ]},
              ],
            }
          }
        },
      });

    </script>
  </body>
</html>

`

**Comment by rcloss** [comment]
Looks like while downgrading to a working version helps, if you have a template saved that was created with the newer mjml plugin the stylemanager parameters still don't reflect in the sidebar properly for those blocks that were added with the more recent version of the mjml plugin

**Comment by aakash2610** [comment]
I am also facing the same issue, and I am using v0.5.5. Is there any workaround (other than downgrading) or timeline for fixing this issue?

**Comment by DRoet** [comment]
resolved in `v0.5.6`

**Comment by aakash2610** [comment]
@DRoet, I upgraded to v0.5.6 but still facing the same issue.
